test/basicfuns.jl

```diff
@@ -176,8 +176,18 @@ end
 end
 
 @testset "logsumexp" begin
-    @test logaddexp(2.0, 3.0)     ≈ log(exp(2.0) + exp(3.0))
-    @test logaddexp(10002, 10003) ≈ 10000 + logaddexp(2.0, 3.0)
+    for T1 in (Int, Float32, Float64), T2 in (Int, Float32, Float64)
```

Comment:
```suggestion
    Ts = (Int, Float32, Float64)
    for T1 in Ts, T2 in Ts
```
(in case we want to add types later)